sentences/sv/cover_HassTurnOff.yaml

```diff
@@ -4,7 +4,7 @@ intents:
     data:
       - sentences:
           - (regla | <stäng_gardiner>) garagedörren
-        response: cover_device_class
+        response: cover
```

Comment:
This should still response `Öppande garaget` (depending on the state of translation it might give the device_class in that case we might need to handle the translation also.